protobuf.sh

```diff
@@ -4,34 +4,21 @@ source: https://github.com/protocolbuffers/protobuf
 build_requires:
   - CMake
   - "GCC-Toolchain:(?!osx)"
+  - alibuild-recipe-tools
+prepend_path:
+  # The protobuf headers must match the protoc binary version, so prevent the
+  # use of system headers by putting ours first in the path.
+  PKG_CONFIG_PATH: "$PROTOBUF_ROOT/lib/pkgconfig"
 ---
-
-cmake $SOURCEDIR/cmake                  \
-    -DCMAKE_INSTALL_PREFIX=$INSTALLROOT \
-    -Dprotobuf_BUILD_TESTS=NO           \
-    -Dprotobuf_MODULE_COMPATIBLE=YES    \
-    -Dprotobuf_BUILD_SHARED_LIBS=OFF    \
+#!/bin/bash -e
```

Comment:
For the record, this is not actually needed nor used...

Reply:
You're right, we run the script with `bash -e <script>` explicitly, so the shebang line doesn't matter. I only find it helps to make the run conditions obvious, and it helps tools like shellcheck figure out that this is how the script is invoked.